features/hsl.yml

```diff
@@ -0,0 +1,13 @@
+name: HSL
+description: The HSL color model specifies colors using hue, saturation and lightness channels. It can be specified using `hsl()` and `hsla()` functions.
+spec: https://drafts.csswg.org/css-color-4/#the-hsl-notation
+caniuse: css3-colors
+group: css
+status:
+  compute_from: css.types.color.hsl
+# Editorial override has been applied to ignore mixed_type_parameters.
```

Comment:
It's unclear why. Is this the same reason as the Oklab feature? If so, I'd suggest copying the whole comment here (including the key word TODO). Otherwise, it'd be nice to more about this to figure out how to proceed.